src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -52,7 +52,7 @@
   private static final String[] STUN_SERVERS = {
       "numb.viagenie.ca", "stun.ekiga.net", "stun.gmx.net", "stun.1und1.de"};
   private static final int STUN_SERVER_PORT = 3478;
-  private static final int LOCAL_FORWARDING_PORT = 2601;
+  private static final int PROXY_PORT = 2601;
```

Comment:
What about giving the Property below a default value, and remove this one? i think it is unneeded and can distract, since the Property can be changed an this one cannot... Also PackageSize and StunServerPort